docs/learn/learn-consensus.md

```diff
@@ -124,7 +124,8 @@ For more details on BABE, please see the
 
 ### BADASS BABE: SASSAFRAS
 
-BADASS BABE is an extention of BABE and acts as a constant-time block production protocol.
+Stands for Semi Anonymous Sortition of Staked Assignees For Fixed-time Rhythmic Assignment of Slots. 
+It is an extention of BABE and acts as a constant-time block production protocol.
```

Comment:
```suggestion
(aka SASSY BABE or BADASS BABE), is an extension of BABE and acts as a constant-time block production protocol.
```

Reply:
I'm unsure about the origins of SASSY BABE or BADASS BABE, but if they are appropriate, please include them